DAT_files/bd+56d524.dat

```diff
@@ -7,5 +7,9 @@ H = 9.000 +/- 0.030
 K = 8.970 +/- 0.020
 IUE = IUE_Data/bd+56d524_iue.fits
 FUSE = FUSE_Data/bd+56d524_fuse.fits
+SpeX_SXD = SpeX_Data/bd+56d524_SXD_spex.fits
+SpeX_LXD = SpeX_Data/bd+56d524_LXD_spex.fits
 sptype = B1V
 uvsptype = B0.5V
+corfac_spex_SXD = 0.9849831324848323
```

Comment:
The number of significant figures is too many.  Maybe just keep 3?  That would give a better idea of the significance of the correction.

Reply:
Ok, I will update this.